lib/blockscore/actions/update.rb

```diff
@@ -27,20 +32,39 @@ module Update
 
       def_delegators 'self.class', :endpoint, :patch
 
+      # May be deprecated in future releases, prefer :update.
       def save!
         if persisted?
-          patch("#{endpoint}/#{id}", filter_params)
+          patch(update_url, filter_params)
           true
         else
           super
         end
       end
 
+      def update(params)
+        update!(params)
+        true
+      rescue
+        false
+      end
+
+      def update!(params)
+        response = patch(update_url, params)
+        self.attributes = response.attributes
+      end
```

Comment:
There is also a [save!](https://github.com/BlockScore/blockscore-ruby/blob/master/lib/blockscore/base.rb#L50) and `ActiveRecord` has a `save` and `save!`, so it was for consistency purposes, but I'm not attached to it.